f5router/f5router.go

```diff
@@ -456,16 +456,32 @@ func (r *F5Router) makeRouteRule(ru updateHTTP) (*bigipResources.Rule, error) {
 	}
 
 	var c []*bigipResources.Condition
-	if true == strings.HasPrefix(ru.URI().String(), "*.") {
-		c = append(c, &bigipResources.Condition{
-			EndsWith: true,
-			Host:     true,
-			HTTPHost: true,
-			Name:     "0",
-			Index:    0,
-			Request:  true,
-			Values:   []string{strings.TrimPrefix(u.Host, "*")},
-		})
+	if strings.Contains(ru.URI().String(), "*") {
```

Comment:
Should we add a check here if there are multiple wildcards?

Reply:
It seems that the below code handles that case by using Split and numSplits so probably unnecessary to add an additional count to this - just existence is enough.